### PR TITLE
Replace isNestedProperties with isStandardSyntaxRule in no-descending-specificity rule 

### PR DESCRIPTION
**Pull request by brneto**
> Which issue, if any, is this issue related to?

Issue #3283

> Is there anything in the PR that needs further explanation?

Refactor PR #3283 accordingly with the suggestion [here](https://github.com/stylelint/stylelint/pull/3283#issuecomment-385763788).

### CODE REVIEW

File: lib/rules/no-descending-specificity/index.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ const rule = function(actual) {
 
     root.walkRules(rule => {
       // Ignore custom property set `--foo: {};` and nested property `foo: {};`
-      if (isCustomPropertySet(rule) || isNestedProperty(rule)) {
+      if (isCustomPropertySet(rule) || !isStandardSyntaxRule(rule)) {
```
Comment (jeddy3) [inline lib/rules/no-descending-specificity/index.js]: A very minor request; can we have this as a separate statement, please?. This is the convention generally used [else where in the code base](https://github.com/brneto/stylelint/blob/5f1f00ce2d148dd56d0df9bedf16d77ea38072b7/lib/rules/property-no-unknown/index.js#L45-L65)

Comment (brneto) [reply lib/rules/no-descending-specificity/index.js]: Of course!

**Comment by brneto** [comment]
I've made a little mistake at last commit title! I've wrote `convections`, but I meant `conventions`! :grimacing:

**Comment by jeddy3** [comment]
@brneto Thanks for making the change!

> I've made a little mistake at last commit title! I've wrote convections, but I meant conventions! 

No worries :) FYI, you can [rewrite commit history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History), but don't worry about doing it this time. As we'll squash the commits down and rewrite then during the merge process instead.

**Comment by brneto** [comment]
@jeddy3 Are you not planning to merge this PR to the next version?

**Comment by jeddy3** [comment]
> @jeddy3 Are you not planning to merge this PR to the next version?

Oops. There are quite a few moving parts at the moment. I've been so focused on getting that regression fix out that I missed this. I released just before you wrote.

No biggy, though. This PR just a refactor. Your original PR fixed the issue and those the changelog is correct.

**Comment by brneto** [comment]
Ok! I was just wondering if would have a reason for this PR haven’t been merged! Thanks!

**Pull request merged**